### PR TITLE
5.3 release notes as of preview 3

### PR DESCRIPTION
**Pull request by rrelyea**

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[f28c3a8](https://github.com/NuGet/docs.microsoft.com-nuget/commits/f28c3a8f714fbf2ebf513b2f3c8e89a408613adb)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.3.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.3/docs/release-notes/NuGet-5.3.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.3?branch=pr-en-us-1659) | [Details](#user-content-5b18b9cc-624ef8d0)

<a id="5b18b9cc-624ef8d0"></a>
### [docs/release-notes/NuGet-5.3.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.3/docs/release-notes/NuGet-5.3.md)
 - **[Warning]** `````Missing attribute: title. Add a title string to show in search engine results.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C4%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201909042041572864-1659%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ZdmA%2BHVUdO37EPxsJzKXFlwCxzQflf0D6urXbN3um74%3D&st=2019-09-04T20%3A38%3A01Z&se=2019-10-05T20%3A43%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/release-notes/NuGet-5.3.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+[Full Changelog]()
+
+[Issues List](https://github.com/nuget/home/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%225.3")
+
+**DCR:**
+
+* Update Xamarin frameworks to map to NetStandard 2.1 - [#8368](https://github.com/NuGet/Home/issues/8368)
+
+* Enable copying the contents of package manager "preview window" for install/update - [#8324](https://github.com/NuGet/Home/issues/8324)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-5.3.md]: Maybe feature?

File: docs/release-notes/NuGet-5.3.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+* Enable restore on .proj files - [#8212](https://github.com/NuGet/Home/issues/8212)
+
+* Introduce `NUGET_NETFX_PLUGIN_PATHS` and `NUGET_NETCORE_PLUGIN_PATHS` to support configuration of both at same time - [#8151](https://github.com/NuGet/Home/issues/8151)
+
+* Enable multiple versions for a PackageDownload via Version attribute - [#8074](https://github.com/NuGet/Home/issues/8074)
+
+* Add -SolutionDirectory and -PackageDirectory options to nuget.exe pack - [#7163](https://github.com/NuGet/Home/issues/7163)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-5.3.md]: Feature maybe?

File: docs/release-notes/NuGet-5.3.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
+#5.3 Release Notes
+
+[Full Changelog]()
```
Comment (nkolev92) [inline docs/release-notes/NuGet-5.3.md]: Can we include the SDK version table?

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[cfc015c](https://github.com/NuGet/docs.microsoft.com-nuget/commits/cfc015ca6d6737dd90abb9cde8b87c547b63a940)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.3.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.3/docs/release-notes/NuGet-5.3.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.3?branch=pr-en-us-1659) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C6%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201909061739107597-1659%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=LAavH9uErN4CYJ7LGxQCt47SFOTru4Vyfor7nSVLbog%3D&st=2019-09-06T17%3A35%3A10Z&se=2019-10-07T17%3A40%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[184cc7a](https://github.com/NuGet/docs.microsoft.com-nuget/commits/184cc7abb41407db11b8e655289e5ebd720d7655)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.3/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.3?branch=pr-en-us-1659) |
[docs/release-notes/NuGet-5.3.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.3/docs/release-notes/NuGet-5.3.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.3?branch=pr-en-us-1659) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C6%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201909061746046887-1659%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2FpfmWNGfWPOshRaJc01C92YiHQhWVzT7g3EGEbq06bQ%3D&st=2019-09-06T17%3A42%3A12Z&se=2019-10-07T17%3A47%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/release-notes/NuGet-5.3.md (hunk 3 above)
Comment (karann-msft) [reply docs/release-notes/NuGet-5.3.md]: Added. But need help here.

File: docs/release-notes/NuGet-5.3.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+ms.topic: conceptual
+---
+
+# NuGet 5.3 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version| Available in .NET SDK(s)|
+|:---|:---|:---|
+| [**5.3.0-preview3**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.3 Preview 3](https://visualstudio.microsoft.com/vs/preview/) | [2.1.80X](https://dotnet.microsoft.com/download/dotnet-core/3.0)<sup>1</sup>, [2.2.40X](https://dotnet.microsoft.com/download/dotnet-core/2.2)<sup>2</sup>, [3.0.100-preview9](https://dotnet.microsoft.com/download/dotnet-core/3.0) |
```
Comment (nkolev92) [inline docs/release-notes/NuGet-5.3.md]: Only 3.0.100-preview9 applies here.

File: docs/release-notes/NuGet-5.3.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+* Regression in 'Nuget sources add' causes "The ':' character, hexadecimal value 0x3A, cannot be included in a name" errors - [#7948](https://github.com/NuGet/Home/issues/7948)
+
+* NuGet plugin credential providers - hide the process window - [#7511](https://github.com/NuGet/Home/issues/7511)
+
+* Enforce PackagePathResolver is an absolute path - [#7349](https://github.com/NuGet/Home/issues/7349)
+
+* Reduce UI refreshes in install and update tabs of Package Manager UI - [#6570](https://github.com/NuGet/Home/issues/6570)
+
+**DCR:**
```
Comment (nkolev92) [inline docs/release-notes/NuGet-5.3.md]: Why did we merge the DCR and Feature stuff? :D

Comment (karann-msft) [reply docs/release-notes/NuGet-5.3.md]: its not merged. I moved the feature to the top as part of the summary.

File: docs/release-notes/NuGet-5.3.md (hunk 4 above)
Comment (karann-msft) [reply docs/release-notes/NuGet-5.3.md]: thanks.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[800ec0c](https://github.com/NuGet/docs.microsoft.com-nuget/commits/800ec0c90e5e00f15e283f5a2d2223d53c90a9d4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.3/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.3?branch=pr-en-us-1659) |
[docs/release-notes/NuGet-5.3.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.3/docs/release-notes/NuGet-5.3.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.3?branch=pr-en-us-1659) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C6%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201909061823322173-1659%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=WS9qUBuUwRm9qaoKnZHUk5feuovLn25YSJfXvEnWps4%3D&st=2019-09-06T18%3A19%3A44Z&se=2019-10-07T18%3A24%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[06760c5](https://github.com/NuGet/docs.microsoft.com-nuget/commits/06760c5a69095c260ebbed196d66c1e3165c4227)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.3/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.3?branch=pr-en-us-1659) |
[docs/release-notes/NuGet-5.3.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.3/docs/release-notes/NuGet-5.3.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.3?branch=pr-en-us-1659) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C6%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201909061834204331-1659%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=b79483CDSzwFOt7B6EE9Fr95sJy2otudubohUeb%2B3ms%3D&st=2019-09-06T18%3A30%3A24Z&se=2019-10-07T18%3A35%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[7192a83](https://github.com/NuGet/docs.microsoft.com-nuget/commits/7192a833b7470a10a6a22015356784220023bddb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.3/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.3?branch=pr-en-us-1659) |
[docs/release-notes/NuGet-5.3.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.3/docs/release-notes/NuGet-5.3.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.3?branch=pr-en-us-1659) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C6%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201909062149595641-1659%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=e%2FulfhSs%2BeCzJnZ1rCGz5n%2BAeOxahUWfr8%2FfkHOjJmw%3D&st=2019-09-06T21%3A46%3A09Z&se=2019-10-07T21%3A51%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**